### ISSUE TITLE
The function of detach() for output feature VIBE in Regressor_all

### ISSUE THREAD

**Issue by yeminglang** [yanmn/RELI11DandLIER#3]
Dear author, when I read your code, I found that you define two modules, VIBE and VIBE2, in which the output feature of the VIBE module is applied detach() operated, that is, the parameters in VIBE are not optimized. Why is it designed this way? It looks like the features of VIBE need to be fed into VIBE2, as a cascading structure, they don't look like the kind of structure of generators and discriminators, and there is no pre-trained model loaded for this part (VIBE), so I'm confused as to why detach() the output features of VIBE? I hope you can answer my question, thanks.

such as 
```
img_feature, smpl_output = self.vibe(data["img_feature"], pc_feature = pc_feature.detach())
img_feature_detached = self.fc(img_feature).view(B, T, D).detach()

event_feature, smpl_output = self.vibe(data["event_feature"], pc_feature = pc_feature.detach())
event_feature_detached = self.fc(event_feature).view(B, T, D).detach()
```

**Comment by XJTU-Haolin** [comment]
same question

**Comment by yanmn** [comment]
Hi, that's a good question.

1. In our implementation, the VIBE module here refers to one of the modules in the [VIBE](https://github.com/mkocabas/VIBE)framework, adjusted to handle our multimodal inputs (e.g., img_feature and pc_feature). It generates initial SMPL parameters and refined features.
VIBE2 module: This is a subsequent module that receives the output of VIBE (e.g., separated features) and further processes it for refining pose estimation or integrating additional modalities.
Using detach() indicates that the parameters of the VIBE module are not optimized in the current training phase. In our pipeline, VIBE acts as a feature extractor or initial pose estimator, and its weights are already well tuned for RGB pose estimation. By detaching its output, we freeze its parameters and treat it as a fixed component, focusing the optimization on downstream modules (such as VIBE2). These outputs are then fed into VIBE2 or other components for further refinement. Separating the output ensures that the gradients of VIBE2 or subsequent layers do not update VIBE, maintaining its role as a stable base estimator while allowing subsequent modules to adapt to multimodal tasks.
Our LEIR approach emphasizes efficient fusion of LiDAR, RGB, and Event modalities (via TUMM and cross-attention). If VIBE is primarily an RGB-based pose estimator, we may not want to fine-tune it on our dataset, especially since its original training (e.g. on an RGB video dataset) may not generalize directly to our multimodal setting. Instead, we separate its output and optimize the fusion and downstream components (e.g. VIBE2, TUMM, SMPL solver) to exploit all modalities.

2. This is not a generator-discriminator setting. Instead, this is a sequential pipeline where VIBE provides the base and VIBE2 or other layers build on it. The detach() operation simply draws the line between frozen and trainable components.

3. In self.vibe(data["img_feature"], pc_feature=pc_feature.detach()), detaching pc_feature ensures that VIBE's gradients are not propagated back to the LiDAR feature extractor (e.g. PointNet++ and GRU). VIBE takes it as a fixed input to assist RGB-based pose estimation without changing the LiDAR feature computation.

I hope my answer can help you.

**Comment by yeminglang** [comment]
Thanks a lot！
